neptune/new/attributes/series/float_series.py

```diff
@@ -69,3 +70,29 @@ def get_last(self, wait=True) -> float:
             self._experiment.wait()
         val = self._backend.get_float_series_attribute(self._experiment_uuid, self._path)
         return val.last
+
+    def fetch_values(self, include_timestamp=True):
+        # pylint: disable=import-outside-toplevel
+        import pandas as pd
+        limit = 1000
+        val = self._backend.get_float_series_values(self._experiment_uuid, self._path, 0, limit)
+        data = val.values
+        offset = limit
+
+        def make_row(entry: FloatPointValue) -> Dict[str, Optional[Union[str, float, datetime]]]:
```

Comment:
You've implemented `make_row` twice.

The only difference is optional `entry` type: `FloatPointValue` vs `StringPointValue`, which we can somehow connect using subclassing.